components/list/list-item-drag-drop-mixin.js

```diff
@@ -606,7 +596,7 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 
 	_renderDragTarget(templateMethod) {
 		templateMethod = templateMethod || (dragTarget => dragTarget);
-		return this.draggable && !this._keyboardActive ? templateMethod(html`
+		return this.draggable && !this._keyboardActive ? templateMethod.call(this, html`
```

Comment:
Was this causing an issue?

Reply:
Just necessary to add the handlers to the outside control action.

Reply:
I thought `this` would have been the component in either context.  🤷‍♂️